SPECS/azure-iot-sdk-c/azure-iot-sdk-c.spec

```diff
@@ -6,7 +6,7 @@ Name:           azure-iot-sdk-c
 # For apt-get packages they fix the version number to 0.2.0 and increase the release number with each release.
 # Since we want to control the release number as thr distribution, this scheme is not applicable for us.
 # They also used to use a regular versioning scheme like 1.3.7 but they did not tag their latest LTS with a version like that.
-Version:        2023.08.07
+Version:        2024.03.04
```

Comment:
I guess the `1. git clone...` instruction in the comment a few lines below could use an upgrade in the `--branch` argument to reflect the new version. I'm guessing it would be even better to convert the comment into a script the auto-upgrade pipeline can run, if you have the time.